package.json

```diff
@@ -25,8 +25,7 @@
     "postrewrite": "node bin/post-rewrite.js",
     "publish": "rimraf public_html && cp -r dist public_html",
     "serve-static": "ws -d dist/ -s index.html -p 4242",
-    "start": "yarn build:clean && env NODE_ENV=development node server.js",
```

Comment:
Note this isn't a `build` but a `clean`. I believe this is still useful to copy the static assets to be served by the server (see `  contentBase: config.output.path`). Especially `zee-worker.js` is absent and needed (`.htaccess` is unused and we don't use `analytics.js` in dev mode).

It would be a lot better to have a watcher in `server.js` that would copy them at startup and as they change, but since it's not the case at the moment I believe we still need this step `build:clean` before launching the server.